e2e/verify/verify.go

```diff
@@ -289,6 +289,66 @@ func (c *ctx) singularityVerifySigner(t *testing.T) {
 	}
 }
 
+func (c *ctx) checkGroupidOption(t *testing.T) {
+	/*
+		// Create a temp directory where we can store a valid image
+		tempDir, err := ioutil.TempDir("", "")
+		if err != nil {
+			t.Fatalf("failed to create a temporary directory: %s", err)
+		}
+		defer func() {
+			err := os.RemoveAll(tempDir)
+			if err != nil {
+				t.Fatalf("failed to delete temporary directory: %s", err)
+			}
+		}()
+
+		imgPath := filepath.Join(tempDir, "imgTest.sif")
+
+		// Pull a valid image
+		e2e.PullImage(t, c.env, successURL, imgPath)
+	*/
+	cmdArgs := []string{"--groupid", "0", c.successImage}
+	c.env.RunSingularity(
+		t,
+		e2e.WithPrivileges(false),
+		e2e.WithCommand("verify"),
+		e2e.WithArgs(cmdArgs...),
+		e2e.ExpectExit(
+			0,
+			e2e.ExpectOutput(e2e.ContainMatch, "Container is signed by 1 key(s):"),
```

Comment:
Should this be checking the json output? Maybe use a function like `verifyOutput`?

Reply:
I think this is a fine start for these tests since they are not intended to cover all cases

Reply:
this is exactly the intent here :)